### ISSUE TITLE
Update

### ISSUE THREAD

**Issue by kobrin666** [halil-durmaz/VFS_Poland_work_visa_appointment_finder#1]
good afternoon. I need to register the whole family. can you make the program take the data of not one person, but different ones? did they register one, and then take the second, and so on? Thanks

**Comment by halil-durmaz** [comment]
Sorry, I am extremely busy these days. You can clone the repo and add that feature. Have a nice day.

**Comment by kobrin666** [comment]
yes, I'm not in a hurry. it won't work on its own. I wanted to ask you for advice.
also had another question. the code states that it can run all day long. but after a while an error pops up. that ip is blocked for 2 hours. and the code crashes with an error.

**Comment by halil-durmaz** [comment]
Even though it is designed to run all day long, it might crash due to the infrastructure of the website. For example, a few weeks ago, it was impossible to schedule an appointment even though there is an appointment; this issue lasted almost 1 whole week...Finally, they fixed it.

I tried to run the program for around 5 hours, and no problem occurred. Could you tell me the details precisely, so that I can review it when I have free time?

**Comment by kobrin666** [comment]
OK. when an error appears, I will make a screen and send it. Which is better via email or whatsapp?
I also noticed another error, when the load on the site is heavy or the Internet speed drops, the page takes a long time to load and the program crashes due to a small timeout between page transitions, as I think.

**Comment by kobrin666** [comment]
like this
https://prnt.sc/EiSN-j6rPvIZ

**Comment by kobrin666** [comment]
clearing the browser does not help, only changing the ip. today it appeared 3 minutes after launch
